.github/ISSUE_TEMPLATE/open_issue.yml

```diff
@@ -4,10 +4,10 @@ body:
 - type: markdown
   attributes:
     value: |
-      1. Please remember that if you are logging a bug for some service that has *stopped working*, please log the bug [here](https://github.com/getferdi/recipes/issues)
-      2. If you are requesting support for a new service in Ferdi, please log it [here](https://github.com/getferdi/recipes/pulls)
-      3. Please remember to read the [self-help documentation](https://github.com/getferdi/ferdi#troubleshooting-recipes-self-help) - in case it helps you unblock yourself for issues related to old recipes.
+      1. Please remember that if you are logging a bug for some service that has *stopped working*, please log the bug [here](https://github.com/ferdium/recipes/issues)
+      2. If you are requesting support for a new service in Ferdium, please log it [here](https://github.com/ferdium/recipes/pulls)
+      3. Please remember to read the [self-help documentation](https://github.com/ferdium/ferdium#troubleshooting-recipes-self-help) - in case it helps you unblock yourself for issues related to old recipes.
       4. Please fill our form below, this way we can analyse and fix the problem as fast as possible.
-      5. Please consider supporting Ferdi!
-        👉  https://github.com/sponsors/getferdi
-        👉  https://opencollective.com/getferdi/donate
+      5. Please consider supporting Ferdium!
```

Comment:
These links don't exist yet, not a problem but leaving a not for us to fix that in the future

Reply:
commented out in new commit